docs/rules/no-unused-prop-types.md

```diff
@@ -86,7 +86,7 @@ This rule can take one argument to ignore some specific props during validation.
 
 ## Known Issues/Limitations
 
-- [False Positives: SFC (helper render methods)](#false-positives-sfc)
+* [False Positives: SFC (helper render methods)](#false-positives-sfc)
```

Comment:
i actually pretty strongly prefer the `-` to the `*` for list items - this is one of the gross things prettier does, too. can we configure that?

Reply:
Yes, configured and autofixed all of them to that style: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md004---unordered-list-style